mozilla-release/toolkit/mozapps/extensions/nsBlocklistService.js

```diff
@@ -381,6 +383,14 @@ Blocklist.prototype = {
 
   /* See nsIBlocklistService */
   getAddonBlocklistState: function Blocklist_getAddonBlocklistState(addon, appVersion, toolkitVersion) {
+#if 0
+    // It's also possible to block all addons through blocklist.
+    console.log("BLOCKLIST CHECK", addon.id, addon.type);
+    const isCliqzExtension =
+        (addon.type === "extension") && (addon.id === "cliqz@cliqz.com");
```

Comment:
@maxim-cliqz this would not work because anybody could call their addon cliqz@cliqz.com